quantized_mesh_tile/terrain.py

```diff
@@ -529,6 +532,52 @@ def toFile(self, filePath, gzipped=False):
             with gzip.open(filePath, 'wb') as f:
                 self._writeTo(f)
 
+    def _getWorkingUnitLatitude(self):
+        if not self._workingUnitLatitude:
+            self._workingUnitLatitude = old_div(self.MAX, (self._north - self._south))
+        return self._workingUnitLatitude
+
+    def _getWorkingUnitLongitude(self):
+        if not self._workingUnitLongitude:
+            self._workingUnitLongitude = old_div(self.MAX, (self._east - self._west))
+        return self._workingUnitLongitude
+
+    def _getDeltaHeight(self):
+        if not self._deltaHeight:
+            max = self.header['maximumHeight']
+            min = self.header['minimumHeight']
```

Comment:
min and max are reserved in python, maybe adapt it

Reply:
ok